modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-gather-log_{context}"]
+= Gathering log data from a failed Agent-based installation
+
+Use the following procedure to gather data about a failed Agent-based installation to provide for a support case.
+
+.Procedure
+
+. Run the following command and collect its output:
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for bootstrap-complete --log-level=debug
+----
+
+. If the output from the previous command indicates a failure, or if the bootstrap is not progressing, run the following command for each node and collect its output:
++
+[source,terminal]
+----
+$ ssh core@<node-ip> sudo /usr/local/bin/agent-gather -O > <local_tmp_path>/agent-gather.tar.xz
```

Comment:
If you were to gather this data from each node, then all you would change is the <node-ip> value above each time, right?

Reply:
Yes